app/controllers/tattoos_controller.rb

```diff
@@ -27,4 +30,13 @@ def create
   def tattoo_params
     params.require(:tattoo).permit(:title, :description, :url, :studio, :artist)
   end
+
+  def rating_average(reviews)
+    sum = 0.0
+    reviews.each do |review|
+      sum += review.rating
+    end
+    sum = sum / reviews.length
+    sum = sum.round(1)
+  end
```

Comment:
It's not very traditional to have extra helper methods just floating around in your controller. How about you make `rating_average` an instance method for `Tattoo`? Then you could call `@tattoo.rating_average`. Also would make it easier to test!